### PR TITLE
Ccp 1449 add add to cart button

### PR DESCRIPTION
**Pull request by dqdquint**
Used shop 31113 to test and deployed 1.0.0-rc.2 for testing.
Used sdk version 1.8.0 in order to circumvent issue opened in PWA-1934.
Additionally, I could use help in unit testing the AddToCartPicker and sub component.

### CODE REVIEW

File: frontend/locale/en-US.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "product_list_add_to_cart": {
+    "modal": {
+      "message": "This product has options",
```
Comment (aaron-martin) [inline frontend/locale/en-US.json]: This should have a better explanation.
This is what the is shown on the favoirtes lists when you click on the add to cart button for a configurable product. I think this add to cart button on product lists should look the same 
image

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/index.jsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      ))}
+    </List>
+  );
+
+  /**
+   * @param {Object} modalProps Props for modal
+   * @returns {JSX}
+   */
+  modalComponent = modalProps =>
+    (<Sheet {...modalProps} title={`Choose Quantity for ${this.props.productName}`} />);
```
Comment (aaron-martin) [inline frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/index.jsx]: We should be passing a translation to the sheet title.

**Review by aaron-martin — CHANGES_REQUESTED** [review]

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-helmet": "^5.2.0",
+    "react-transition-group": "^2.2.1",
+    "react-hot-loader": "^3.1.3",
+    "react-transform-catch-errors": "^1.0.2"
+  },
+  "dependencies": {
+    "@shopgate-ps/pwa-extension-kit": "^0.6.0"
```
Comment (aaron-martin) [inline frontend/package.json]: I don't think you are using the pwa-extension-kit so this can be removed -- I think

**Review by aaron-martin — CHANGES_REQUESTED** [review]

File: frontend/locale/en-US.json (hunk 1 above)
Comment (dqdquint) [reply frontend/locale/en-US.json]: Updated.

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/index.jsx (hunk 2 above)
Comment (dqdquint) [reply frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/index.jsx]: Updated

File: frontend/package.json (hunk 3 above)
Comment (dqdquint) [reply frontend/package.json]: Removed

File: frontend/locale/en-US.json (hunk 1 above)
Comment (aaron-martin) [reply frontend/locale/en-US.json]: > Updated.

I think we should change the copy for the dismiss button too
image
(sorry)

Comment (dqdquint) [reply frontend/locale/en-US.json]: Gotcha. Updated!

**Review by aaron-martin — APPROVED** [review]

**Pull request merged**